jetcd-core/src/main/java/io/etcd/jetcd/impl/AuthCredential.java

```diff
@@ -85,7 +85,7 @@ private void authenticate(MetadataApplier applier) {
 
         if (!interceptorsChain.isEmpty()) {
             authFutureStub = authFutureStub.withInterceptors(
-                interceptorsChain.toArray(ClientInterceptor[]::new));
+                interceptorsChain.toArray(new ClientInterceptor[0]));
```

Comment:
is this really needed ? afaik it should work on java 9 too

Reply:
it works on java 9, but don't on java 8 this error - 
`error: incompatible types: Array is not a functional interface
                interceptorsChain.toArray(ClientInterceptor[]::new));`

Reply:
ah ok